build.psm1

```diff
@@ -1188,6 +1188,53 @@ function Show-PSPesterError
 
 }
 
+function Test-XUnitTestResults
+{
+    param(
+        [Parameter(Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        [string] $TestResultsFile
+    )
+
+    if(-not (Test-Path $TestResultsFile))
+    {
+        throw "File not found $TestResultsFile"
+    }
+
+    try
+    {
+        $results = [xml] (Get-Content $TestResultsFile)
+    }
+    catch
+    {
+        Write-Error $_
```

Comment:
this should probably be throw or something - if `$results` has a problem then it might look like everything passed since the error will likely be hidden in the log

Reply:
added throw instead of `Write-Error`